# Proc Mapping parameters

proc_mapping:
  # Preprocessing
  preprocessing:
    min_intensity: 0.1
    max_intensity: 1.0
    min_range: 0.1         # m
    max_range: 10.0         # m

  filter:
    general:
      box_size: 0.05       # m 

  # Segmentation
  segmentation:
    buoys:
      cluster_dist: 0.4
      plane_tol: 0.05
      icp_inlier_ratio: 0.1
      z_normal_thres: 0.2
      in_plane_thres: 0.4
      min_area: 0.6
      max_area: 3.5